book-content/chapters/14-configuring-typescript.md

```diff
@@ -0,0 +1,1024 @@
+We've dipped into TypeScript's `tsconfig.json` configuration file a few times in this book. Let's take a deeper look. We won't cover every option in `tsconfig.json` - many of them are old and rarely used - but we'll cover the most important ones.
+
+## Recommended Configuration
+
+To start, here's a recommended base `tsconfig.json` configuration with options appropriate for most applications you're building:
+
+```json
+{
+  "compilerOptions": {
+    /* Base Options: */
+    "skipLibCheck": true,
+    "target": "es2022",
```

Comment:
It's 2024 year already - any reason why you recommend 2022 here instead of 2023?